tools/mkdocs/overrides/main.html

```diff
@@ -8,4 +8,7 @@
 <p>
   If you use a Mend-hosted Renovate App <em>and</em> you have encrypted secrets in your Renovate config, then you must migrate the secrets to the Developer Portal UI. If you do not migrate the secrets, eventually Renovate will stop running on repositories that use the Mend-hosted Renovate App. Please read the <a href="https://github.com/renovatebot/renovate/discussions/33407">announcement about the disabling of Encrypted Secrets in Mend-hosted Renovate apps</a> to learn more.
 </p>
+<p>
+  We released a new major version of Renovate. Read the [Renovate 40 changelog on GitHub](https://github.com/renovatebot/renovate/releases/tag/40.0.0).
```

Comment:
This needs to be a html link instead of a markdown link.

Reply:
Thank you for the report! I'm fixing the bad link with this PR:

- #35899

Feel free to double-check my work. :wink: